### ISSUE TITLE
Mac icon not working with 2.2.5

### ISSUE THREAD

**Issue by macecchi** [nwjs/nw-builder#342]
When using nw-builder 2.2.0, I was able to build my app from the Terminal using the `--macIcns` parameter.

After upgrading to 2.2.5, the option does not work anymore. I can see that the icon is being copied to the Resources folder inside the .app but it's not being used.

**Comment by ncgonz** [comment]
I'm seeing this also.

**Comment by adam-lynch** [comment]
Which version of NW.js are you building for?

**Comment by ncgonz** [comment]
i run nwbuild inside a bash script, but I don't specify which version of nw.js, so i'm always building against latest.  Here is the pertinent part of the bash script: 

`nwbuild -o './../build/'$1 '.' --options.winIco icon.ico --options.macIcns icon.icns --options.zip false -p win32,win64,osx64`

**Comment by adam-lynch** [comment]
Using the [example app (`/example/nwapp` directory)](https://github.com/nwjs/nw-builder/tree/master/example/nwapp), it works both programmatically (i.e. using the example gulpfile) and using the command-line too.

@ncgonz You need to use `--macIcns` not `--options.macIcns`.

I'm going to close. If the problem persists, please give me an example project.

**Comment by fritx** [comment]
`macIcns` doesn't work for me either. nw-builder v2.2.6.
With 2.2.0, it worked fine.

I saw the Info.plist still list `nw.icns`

```plain
<key>CFBundleIconFile</key>
<string>nw.icns</string>
```

**Comment by fritx** [comment]
Here is my snippet.

```js
var nw = new NwBuilder({
  // https://github.com/nwjs/nw-builder/issues/312
  // cacheDir: './node_modules/nw/nwjs',
  files: '../regt-pack/**',
  buildDir: '../regt-app',
  macIcns: '../regt-pack/main/Genius.icns',
  version: '0.12.3',
  platforms: ['osx64']
})
```

**Comment by adam-lynch** [comment]
@fritx does it work for you when using the example?

**Comment by fritx** [comment]
![image](https://cloud.githubusercontent.com/assets/6647633/17648681/d5323424-6250-11e6-8ecf-286cb32f0d5e.png)

Seems still no work. @adam-lynch I'm not able to download nw 0.14.6 and use 0.12.3 instead.

```js
  var nw = new NwBuilder({
        // version: '0.14.6',
        version: '0.12.3',
        files: './nwapp/**',
        macIcns: './icons/icon.icns',
        macPlist: {mac_bundle_id: 'myPkg'},
        platforms: ['osx64']
    });
```

**Comment by adam-lynch** [comment]
It was broken for 0.12.3 (but worked for every version before and after that). Fixed now in 2.2.7. Thanks @fritx
